package.json

```diff
@@ -1,39 +1,37 @@
 {
-  "name": "glsl-pipeline",
-  "version": "1.0.6",
-  "description": "Prototype complex pipelines directly from a single shader by branching it into stages.",
+  "name": "glsl-pipeline-monorepo",
+  "private": true,
+  "version": "1.0.0",
   "main": "index.js",
   "license": "MIT",
   "author": "Patricio Gonzalez Vivo <patriciogonzalezvivo@gmail.com>",
+  "maintainers": [
+    "Amin Shazrin (https://github.com/ammein)"
```

Comment:
add me and @marioecg ( https://github.com/marioecg ) here please. Mario helped a big deal on the first version.